docker-compose.yml

```diff
@@ -23,20 +23,20 @@ services:
         aliases:
           - apps
 
-
   elm:
     build:
       context: .
       dockerfile: Dockerfile-elm
     volumes:
       - "./:/code"
+      - "assets:/code/jarbas/frontend/static"
     command: [npm, run, watch]
   nginx:
     build:
       context: .
       dockerfile: Dockerfile-nginx
     volumes:
-      - "staticfiles:/opt/jarbas/staticfiles"
+      - "assets:/opt/jarbas/staticfiles"
```

Comment:
Sorry, I don't know about nginx, is this line to serve staticfiles from nginx instead of jarbas container?
Are the files from jarbas container still being served?